packages/db-migrations/databases/fxa/patches/patch-137-136.sql

```diff
@@ -0,0 +1,21 @@
+-- -- Delete new row from table
+-- DELETE FROM securityEventNames WHERE name='account.login.failure';
+-- DELETE FROM securityEventNames WHERE name='account.two_factor_added';
+-- DELETE FROM securityEventNames WHERE name='account.two_factor_requested';
```

Comment:
But maybe this is the bug and not the fact we aren't `INSERT`ing above. A quick Ctrl+F on this event name shows no usages in this PR. 🤷

Reply:
This is not part of my change set. Note that this is based on FX1-4924.2